lib/index.js

```diff
@@ -127,7 +127,7 @@ export function testRule(schema) {
 				if (!fix) return;
 
 				assert.ok(
-					fixed || unfixable,
+					fixed || fixed === '' || unfixable,
```

Comment:
[suggest] The `fixed` property should be a `string` or `undefined`, so how about this? 👇🏼 

```suggestion
					typeof fixed === 'string' || unfixable,
```

Reply:
FYI.

https://github.com/stylelint/stylelint-test-rule-node/blob/923e256ff3def1cf6a78e36f6d6bb2d743223045/lib/index.d.ts#L72-L75